src/components/Download/Summary/VersionCheck/index.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+import React from 'react';
+import T from 'prop-types';
+
+import loadData from 'higherOrder/loadData';
+import Link from 'components/generic/Link';
+import Tooltip from 'components/SimpleCommonComponents/Tooltip';
+import { getUrlForMeta } from 'higherOrder/loadData/defaults';
+
+import { foundationPartial } from 'styles/foundation';
+
+import interproTheme from 'styles/theme-interpro.css'; /* needed for custom button color*/
+import ipro from 'styles/interpro-new.css';
+const f = foundationPartial(interproTheme, ipro);
+
+/*:: type Props = {
+  localID: string,
+  downloadVersion: number,
+  data?: {
+    loading: boolean,
+    payload?: {
+      databases:{
+        [string]:{
+          version: number,
+        }
+      }
+    }
+  }
+};*/
+
+const VersionCheck = ({ data, downloadVersion, localID } /*: Props */) => {
+  if (!data || data.loading || !downloadVersion) return null;
+  const currentVersion = Number(
+    data?.payload?.databases?.interpro?.version || 0,
+  );
+  if (currentVersion > downloadVersion)
```

Comment:
The `InterPro-Version-Minor` is only to force to drop the browser cache by a header on the API, which is only the case on new InterPro releases that have some problem in redix. I didn't consider it as it is a bit of an edge case, but I suppose is true someone gets a download minutes after a release and before we notice an issue in redix. DO you think we should cover that case?

Why did you say they are Integers, I tested with `91.1` and it gets the float value. It might be because `Number("90.0") === 90` and `Number("90.1") === 90.1` so I think the cases of "minor" versions are covered.

Reply:
> The `InterPro-Version-Minor` is only to force to drop the browser cache by a header on the API, which is only the case on new InterPro releases that have some problem in redix. I didn't consider it as it is a bit of an edge case, but I suppose is true someone gets a download minutes after a release and before we notice an issue in redix. DO you think we should cover that case?

If it's not too complicated, why not. But to be perfectly honest, I made this comment mostly because, when testing what happens when the versions are different, I got frustrated since it's easier to change the `InterPro-Version` header than the InterPro version in the `/api/` payload. :stuck_out_tongue_closed_eyes: 

> Why did you say they are Integers, I tested with `91.1` and it gets the float value. It might be because `Number("90.0") === 90` and `Number("90.1") === 90.1` so I think the cases of "minor" versions are covered.

My bad. I misread `data?.payload?.databases?.interpro?.version || 0` for `data?.payload?.databases?.interpro?.version | 0` and I thought you were converting string to int using the OR bitwise operator.

Reply:
> using the OR bitwise operator.

I'm not that smart! 😅

Reply:
> when testing 

Actually the testing this time was quite easy for me because my local API is pointing to the `tst` DB in dev, which is already saying Interpro `90.0`